### PR TITLE
Updated code to latest version of upstream and dependencies

### PR DESCRIPTION
**Pull request by lao**
dependencies updated:
- rake
- railties
- activerecord

code is also updated to latestest in upstream

### CODE REVIEW

File: departure.gemspec [hunk 1]
```diff
@@ -23,8 +23,8 @@ Gem::Specification.new do |spec|
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency 'railties', *Array(RAILS_DEPENDENCY_VERSION)
-  spec.add_runtime_dependency 'activerecord', *Array(RAILS_DEPENDENCY_VERSION)
+  spec.add_runtime_dependency 'railties', '>=6.1.5.1'
```
Comment (pcriv) [inline departure.gemspec:26]: Why not make this change on the `RAILS_DEPENDENCY_VERSION` ? then we could make an PR to the gem and upstream this changes

**Review by pcriv — APPROVED** [review]

**Review by linkyndy — COMMENTED** [review]
It's pretty hard to follow which are the updates you pulled and which are the ones you did. From what I get, there isn't much you added on top of the latest departure code and updating the dependencies, correct?

**Comment by pcriv** [comment]
@lao any change we could split this PR in updating from upstream and the actual changes you did?

**Comment by lao** [comment]
> It's pretty hard to follow which are the updates you pulled and which are the ones you did. From what I get, there isn't much you added on top of the latest departure code and updating the dependencies, correct?

Exactly. All I did was updating the dependecies. The problem with the upstream is that this Repo seems to be a copy not a fork of the original. I am planning to update the upstream this change to departure but I will create a fork on my personal account and create a PR from there. This PR here was just a way to fix the security issues quicker.

**Comment by lao** [comment]
> @lao any chance we could split this PR in updating from upstream and the actual changes you did?

This is all I did: https://github.com/WeTransfer/departure/pull/5/commits/0d0c9714d47d04adc0f6b899d994e814067f4707

Due to the fact this is repo is not a real fork, and I did this just for us to fix the security issue quicker, would you mind if I do this work only once in a real forked branch and so I could open a PR and upstream it to the real gem? 

Once we have updated the gem, this branch and this entire repo will be removed. =)

**Comment by lao** [comment]
@pcriv and @linkyndy I will close this PR. 

Apparently the problem was long fixed in the `departure` 's repo and in their master branch is all fixed but for some reason they did not realease a new gem. So I will basically change the gem github address from `weTransfer/departure` to `departurerb/departure` and whenever they release a new gem version we will just use it.

**Comment by linkyndy** [comment]
Wouldn't that mean the security fixes will have to wait til then?

**Comment by lao** [comment]
@linkyndy actually, what happened was, when Pablo approved the changes last week, I was going to merge it to master/main but then I thought that maybe someone else is linking their code or using this main or later someone could actually update the main by mistake, revert. So I decided to create a branch 6.3.1 and use that in storm, with these fixes.

**Review by pcriv — APPROVED** [review]
